atmosphere/settings/local.py.j2

```diff
@@ -303,6 +307,9 @@ AUTHENTICATION_BACKENDS = (
     # For Service-Access
     'django_cyverse_auth.authBackends.LDAPLoginBackend',
     {% endif %}
+    {% if AUTH_ENABLE_OPENSTACK -%}
+    'django_cyverse_auth.authBackends.OpenstackLoginBackend',
+    {% endif -%}
```

Comment:
Reminder: Documenting the new variables, what is expected and ensuring "the secrets repos" have the updated variables pre-merge.